x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elastic/authorization/AuthorizationTaskExecutor.java

```diff
@@ -166,7 +170,14 @@ private boolean shouldSkipCreatingTask(@Nullable ClusterState state) {
             return true;
         }
 
-        return clusterCanSupportFeature(state) == false || running.get() == false || authorizationTaskExists(state);
+        return clusterCanSupportFeature(state) == false
+            || running.get() == false
+            || authorizationTaskExists(state)
+            || ccmSupportedButNotYetConfigured();
```

Comment:
If we're in an environment where ccm is support but the user hasn't provided the api key, then there's no point in creating the polling task because the polls won't succeed until we have an api key.